app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = current_user.comments.new(comment_params)
+    @comment.post_id = params[:post_id]
+
+    if @comment.save
+      redirect_to user_post_path(current_user, @comment.post)
+    else
+      render :create
+    end
+  end
+end
```

Comment:
- [ ] I think there is an issue in your schema when I tried to create for example a new user some attributes are missing in the Database. I would like strongly recommend working on your database and adding new migrations to update to let the user be able to create a new user and a new post and so on. please check this [article](https://stackoverflow.com/questions/1992019/how-can-i-rename-a-database-column-in-a-ruby-on-rails-migration). Please fix to make your app work as expected :+1: